### PR TITLE
WIP Final mobile

### PR DESCRIPTION
**Pull request by Yar3k**
Editing desing for final version and fixing some of bugs.

### CODE REVIEW

**Comment by EvalVis** [comment]
I think you will need to revert commit 
bb29519 (Connect Bot...) because it was duplicate, I fixed it, but you checked out before my final fix.

**Comment by Yar3k** [comment]
> I think you will need to revert commit
> [bb29519](https://github.com/Boulderis/Locals-Trade/commit/bb2951954999fa46bf9012e2410f542fef9e6297) (Connect Bot...) because it was duplicate, I fixed it, but you checked out before my final fix.

I think that it will be not included, because it fixed in main, so basically it will do same here automatically

**Comment by Yar3k** [comment]
And its done for pull!🎉

File: Support_Your_Locals/Controllers/AuthController.cs [hunk 1]
```diff
@@ -79,7 +79,7 @@ public async Task<ActionResult> SignIn(UserLoginModel login)
                         {
                             new Claim(ClaimTypes.Name, user.Name),
                             new Claim(ClaimTypes.NameIdentifier, user.UserID.ToString()),
-                            new Claim(ClaimTypes.Role, isAdmin(login.Email) ? "Administrator": "User"),
+                            //new Claim(ClaimTypes.Role, isAdmin(login.Email) ? "Administrator": "User"),
```
Comment (EvalVis) [inline Support_Your_Locals/Controllers/AuthController.cs]: Uncomment this one out before merging.

File: LocalsTradeBot/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
-  "$schema": "http://json.schemastore.org/launchsettings.json",
   "iisSettings": {
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:3978",
-      "sslPort": 0
+      "applicationUrl": "http://localhost:63904/",
+      "sslPort": 44391
```
Comment (EvalVis) [inline LocalsTradeBot/Properties/launchSettings.json:7]: Ours is 5000 for http and 5001 for https. SSLPort 44311.

File: Support_Your_Locals/Controllers/AuthController.cs [hunk 3]
```diff
@@ -78,7 +78,7 @@ public async Task<ActionResult> SignIn(UserLoginModel login)
                         {
                             new Claim(ClaimTypes.Name, user.Name),
                             new Claim(ClaimTypes.NameIdentifier, user.UserID.ToString()),
-                            //new Claim(ClaimTypes.Role, isAdmin(login.Email) ? "Administrator": "User"),
+                            new Claim(ClaimTypes.Role, isAdmin(login.Email) ? "Administrator": "User"),
```
Comment (EvalVis) [inline Support_Your_Locals/Controllers/AuthController.cs]: OK, I see you uncommented this.

File: Support_Your_Locals/Views/Shared/TimePicker.cshtml [hunk 4]
```diff
@@ -46,18 +48,4 @@
             });
         }
     });
-</script>
-
-<script>
-    document.addEventListener('DOMContentLoaded',
-        function () {
-            var from = document.getElementById("@pickerFrom");
```
Comment (EvalVis) [inline Support_Your_Locals/Views/Shared/TimePicker.cshtml:54]: Where this one has used, and why it is out?

Comment (EvalVis) [reply Support_Your_Locals/Views/Shared/TimePicker.cshtml:54]: OK, I think I got it. But why remove it?

**Review by EvalVis — APPROVED** [review]
I will check it fully, when it is loaded on the web, but it seems good.

**Review by MartynaUb — APPROVED** [review]
Approved, everything looks good

**Pull request merged**